frontend/src/pages/AccountSend.vue

```diff
@@ -866,6 +866,22 @@ function useSendForm() {
     return true;
   }
 
+  function isValidBatchSendAmount(val: string | undefined, tokenInput?: TokenInfoExtended | null | undefined) {
+    const tokenToUse = tokenInput || token.value;
+    if (!tokenToUse) return tc('Send.select-a-token');
+
+    const { address: tokenAddress, decimals } = tokenToUse;
+
+    // Check total Batch Send amount
+    const totalBatchSendAmount = summaryAmount.value.get(tokenToUse);
+    if (totalBatchSendAmount) {
```

Comment:
`totalBatchSendAmount` is a string, so this feels like an odd way to handle the conditional since `Boolean('0')` is true but if the amount is 0 we can skip this block. Maybe convert to BigNumber first and check that it's greater than 0?

Reply:
It was there to check that `totalBatchSendAmount` isn't undefined. `parseUnits` below only takes in strings.
I've updated the check to     
```
if (totalBatchSendAmount && BigNumber.from(totalBatchSendAmount).gt(0)) {
```
in https://github.com/ScopeLift/umbra-protocol/pull/597/commits/218b5c6c5171bc9b8e044c8c049f2d57cddb495c